packages/py-api/py_api/middleware/auth.py

```diff
@@ -23,7 +23,7 @@
 class AuthMiddleware:
     """Utility class for easily initializing all authentication middleware"""
     _BYPASSED_ENDPOINTS: Final = {
-        "/health": ["GET"], "/routes": ["GET"], "/fswitches": ["GET"],
+        "/health": ["GET"], "/routes": ["GET"], "/fswitches": ["GET"], "/hackathon/verify": ["GET"],
```

Comment:
Could you see if you can expose the openapi schema endpoint as well?!

Reply:
More work needs to be done to integrate swagger